digdag-standards/src/main/java/io/digdag/standards/command/SimpleCommandStatus.java

```diff
@@ -0,0 +1,46 @@
+package io.digdag.standards.command;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.digdag.spi.CommandStatus;
+
+class SimpleCommandStatus
+        implements CommandStatus
+{
+    static CommandStatus of(final String ioDirectory, final Process p)
```

Comment:
this is very minor but can we have `Process` as the first argument...? because constructor has uses `statusCode, ioDirectory` order but `of` is opposite

Reply:
Sounds reasonable. Fixed https://github.com/treasure-data/digdag/pull/835/commits/ec1b5a8d25184974d24d115e41d1f1d90b689679